mpxe/integration_tests/test_can_dump.py

```diff
@@ -0,0 +1,15 @@
+import unittest
+import time
+
+from mpxe.integration_tests import int_test
+
+class TestCanDump(int_test.IntTest):
+    def setUp(self):
+        super(TestCanDump, self).setUp()
```

Comment:
In python 3 you can just do `super().setUp()`, this goes for the other super() calls too